CONTRIBUTING.rst

```diff
@@ -6,7 +6,7 @@ Reporting bugs and asking questions
 
 You can post questions or issues or feedback through the following channels:
 
-1. `Github Discussions`_: For discussions about development, questions about usage, and feature requests.
+1. `Discourse forum`_: For discussions about development, questions about usage, and feature requests.
```

Comment:
Let's remove `, and feature requests`. Feature requests goes to github issues.

Reply:
Done.